deploy-manage/deploy/cloud-enterprise/ece-install-offline-with-registry.md

```diff
@@ -43,29 +42,27 @@ Installing ECE on multiple hosts with your own registry server is simpler, becau
 
     ```sh
-    docker tag docker.elastic.co/cloud-enterprise/elastic-cloud-enterprise:3.8.1 REGISTRY/cloud-enterprise/elastic-cloud-enterprise:3.8.1
-    docker tag docker.elastic.co/cloud-assets/elasticsearch:7.17.27-0 REGISTRY/cloud-assets/elasticsearch:7.17.27-0
-    docker tag docker.elastic.co/cloud-assets/kibana:7.17.27-0 REGISTRY/cloud-assets/kibana:7.17.27-0
-    docker tag docker.elastic.co/cloud-assets/apm:7.17.27-0 REGISTRY/cloud-assets/apm:7.17.27-0
-    docker tag docker.elastic.co/cloud-assets/enterprise-search:7.17.27-0 REGISTRY/cloud-assets/enterprise-search:7.17.27-0
-    docker tag docker.elastic.co/cloud-release/elasticsearch-cloud-ess:8.17.1 REGISTRY/cloud-release/elasticsearch-cloud-ess:8.17.1
-    docker tag docker.elastic.co/cloud-release/kibana-cloud:8.17.1 REGISTRY/cloud-release/kibana-cloud:8.17.1
-    docker tag docker.elastic.co/cloud-release/elastic-agent-cloud:8.17.1 REGISTRY/cloud-release/elastic-agent-cloud:8.17.1
-    docker tag docker.elastic.co/cloud-release/enterprise-search-cloud:8.17.1 REGISTRY/cloud-release/enterprise-search-cloud:8.17.1
+    docker tag docker.elastic.co/cloud-enterprise/elastic-cloud-enterprise:4.0.0 REGISTRY/cloud-enterprise/elastic-cloud-enterprise:4.0.0
+    docker tag docker.elastic.co/cloud-assets/elasticsearch:8.18.0 REGISTRY/cloud-release/elasticsearch-cloud-ess:8.18.0
+    docker tag docker.elastic.co/cloud-assets/kibana:8.18.0 REGISTRY/cloud-release/kibana-cloud:8.18.0
+    docker tag docker.elastic.co/cloud-assets/apm:8.18.0 REGISTRY/cloud-release/elastic-agent-cloud:8.18.0
+    docker tag docker.elastic.co/cloud-assets/enterprise-search:8.18.0 REGISTRY/cloud-release/enterprise-search-cloud:8.18.0
+    docker tag docker.elastic.co/cloud-release/elasticsearch-cloud-ess:9.0.0 REGISTRY/cloud-release/elasticsearch-cloud-ess:9.0.0
+    docker tag docker.elastic.co/cloud-release/kibana-cloud:9.0.0 REGISTRY/cloud-release/kibana-cloud:9.0.0
+    docker tag docker.elastic.co/cloud-release/elastic-agent-cloud:9.0.0 REGISTRY/cloud-release/elastic-agent-cloud:9.0.0
     ```
 
 4. Push the Docker images to your private Docker registry, using the same tags from the previous step. Replace `REGISTRY` with your actual registry URL, for example `my.private.repo:5000`:
 
     ```sh
-    docker push REGISTRY/cloud-enterprise/elastic-cloud-enterprise:3.8.1
-    docker push REGISTRY/cloud-assets/elasticsearch:7.17.27-0
-    docker push REGISTRY/cloud-assets/kibana:7.17.27-0
-    docker push REGISTRY/cloud-assets/apm:7.17.27-0
-    docker push REGISTRY/cloud-assets/enterprise-search:7.17.27-0
-    docker push REGISTRY/cloud-release/elasticsearch-cloud-ess:8.17.1
-    docker push REGISTRY/cloud-release/kibana-cloud:8.17.1
-    docker push REGISTRY/cloud-release/elastic-agent-cloud:8.17.1
-    docker push REGISTRY/cloud-release/enterprise-search-cloud:8.17.1
+    docker push REGISTRY/cloud-enterprise/elastic-cloud-enterprise:4.0.0
+    docker push REGISTRY/cloud-assets/elasticsearch:8.18.0
```

Comment:
These also still reference the wrong image

Reply:
fixed in f64a76a1